playground/src/pages/CodingEnvironment.tsx

```diff
@@ -197,8 +207,13 @@ export default function CodingEnvironment(): JSX.Element {
   const verifyProgram = async () => {
     init().then(() => {
       disableDebug();
+      if (!proof) {
```

Comment:
instead of showing an error to the user, we should disable the `Verify` button, and only enable it one a proof is populated

Reply:
good point

Reply:
done